src/runtime/HalidePyTorchHelpers.h

```diff
@@ -48,6 +48,31 @@ inline void check_type(at::Tensor &tensor) {
     AT_ERROR("Scalar type ", tensor.scalar_type(), " not handled by Halide's PyTorch wrapper");
 }
 
+// TODO: if PyTorch exposes any variable with the API version,
+// I haven't found it in source or documentation; for now, we'll sniff
+// this macro's existence to infer that we are building with v1.3+ (vs 1.2)
+#ifdef AT_FORALL_SCALAR_TYPES_WITH_COMPLEX_AND_QINTS
```

Comment:
This is unfortunate but seems unavoidable.

Reply:
I don't know of a better solution, either.